spec/lib/bolt_spec/integration.rb

```diff
@@ -28,8 +28,9 @@ def run_cli(arguments, rescue_exec: false)
       output.string
     end
 
-    def run_cli_json(arguments, **opts)
-      output = run_cli(arguments, **opts)
+    def run_cli_json(arguments, *opts)
```

Comment:
We still want to double splat in the signature so we can accept nil or a hash
We don't need to double splat the opts we can just pass them on as a single hash

```
def run_cli_json(arguments, **opts)
  output = run_cli(arguments, opts)
```

Reply:
Ahhh, I see. I was testing this one and I thought that when I did that it failed. I just re-ran locally and it passed. Makes sense.